components/accordion/index.css

```diff
@@ -72,6 +74,12 @@
 		var(--mod-accordion-item-header-top-to-text-space, var(--spectrum-accordion-item-header-top-to-text-space)) + var(--mod-accordion-item-header-bottom-to-text-space, var(--spectrum-accordion-item-header-bottom-to-text-space)) + (var(--mod-accordion-item-header-font-size, var(--spectrum-accordion-item-header-font-size)) * var(--mod-accordion-item-header-line-height, var(--spectrum-accordion-item-header-line-height)))
 	);
 
+	/* Calculated vertical spacing for action button and switch to center them within the accordion item */
+	--spectrum-accordion-item-direct-actions-vertical-spacing: calc(
+		(var(--mod-accordion-item-min-block-size, var(--spectrum-accordion-item-min-block-size)) -
+		var(--mod-accordion-item-direct-actions-height, var(--spectrum-accordion-item-direct-actions-height))) / 2
+	);
+
```

Comment:
Did I miss a better way to do vertical spacing for this situation? I was finding that this got pretty complex, pretty fast:

* An `align-items: center` on the `.spectrum-Accordion-itemDirectActions` isn't ideal when the text wraps because the designs show that we want direct actions to align with the first line of text, instead of all of them.
* Accordion items have a min-height of `component-height-75`/`component-height-100`/`component-height-200`/`component-height-300`, same with the direct actions, but we don't usually see that min-height in accordion (except for compact) because of the calculations for spacing and font-size * line-height making it larger